### ISSUE TITLE
add content for keras

### ISSUE THREAD

**Issue by rogovski** [rogovski/rogocorp-content#3]
create a content entry for keras.io

### create explanation and description

should be straightforward.

### document project architecture

__Training Pipeline__:

* Establish how users *build* and *request execution of* a training job.
* Establish how to read data into system for training. Is training data stored on the local filesystem, remote filesystem, mongo, s3, ... ? Is training data accessed via a REST api, ftp, zmq, ... ?
* Establish best practices for model checkpointing, training termination conditions (e.g. test accuracy threshold exceeded, loss not decreasing).
* log batch epochs with RemoteMonitor class.

__Prediction Pipeline__

* Establish how users request a classification job with a particular model and input data.
* Establish how to read data into system for a prediction request. 
* Establish how to get data out of the system.
